### PR TITLE
Full angular distribution for Lb -> Lc (->Lambda pi) l nubar for m_l finite case and cvl,cvr,csl,csr NP WCs

### PR DESCRIPTION
**Pull request by akokulu7**

### CODE REVIEW

**Comment by akokulu7** [comment]
@dvandyk could you please review this PR and merge afterwards? thanks!

File: eos/parameters/polarisations.yaml [hunk 1]
```diff
@@ -5,6 +5,13 @@
     max:     +0.655
     comments: "cf. [PDG2012+]"
 
+# Decay parameter of the Lambda_c baryon from FOCUS collaboration
+"Lambda_c::alpha@FOCUS" :
+    central: -0.780
+    min:     -0.986
+    max:     -0.574
+    comments: "cf. [hep-ex/0509042]"
```
Comment (dvandyk) [inline eos/parameters/polarisations.yaml]: @akokulu7 I'd prefer if you used the EOS internal referencing. Also, PDG completely suffices here.

File: eos/parameters/polarisations.yaml [hunk 2]
```diff
@@ -5,6 +5,13 @@
     max:     +0.655
     comments: "cf. [PDG2012+]"
 
+# Decay parameter of the Lambda_c baryon from FOCUS collaboration
+"Lambda_c::alpha@FOCUS" :
```
Comment (dvandyk) [inline eos/parameters/polarisations.yaml]: Comparing to the alpha parameter in hadronic Lamba decays, you appended ```@FOCUS```. I don't see why we need that? Convince me!

**Comment by dvandyk** [comment]
The commit message's title  for the first 2 commits is excessively long. Can you shorten it please? I.e., the m_l = 0 and NP WCs parts can be moved into the body of the message.

File: eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+            double alpha;
+        };
+
+        struct AngularObservables
+        {
+            std::array<double, 10> _k;
+
+            AngularObservables(const Amplitudes & a)
+            {
+                // extend to include full basis
```
Comment (dvandyk) [inline eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc]: Is this comment still relevant? I.e., are there further observables beyond the 10 implemented here?

File: eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+        }
+
+        lambdab_to_lambdac_l_nu::Amplitudes amplitudes(const double & s)
+        {
+            lambdab_to_lambdac_l_nu::Amplitudes result;
+
+            // define below the b->c WCs and the translation rule from BFvD2014 to EOS basis
+            const WilsonCoefficients<BToC> wc = model->wilson_coefficients_b_to_c(opt_l.value(), false);
+
+            const complex<double> c9        = wc.cvl() / 2.0;
```
Comment (dvandyk) [inline eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc]: Ok, this I don't like. It's the easy way out. Please remove any mention of c9, c10 and so on from this file and replace them with cVL, cVR and so on.

File: eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+            const double H0Am1hm1h = -H0Ap1hp1h;
+            const double HpAm1hp1h = -ffpA * std::sqrt(2) * sqrtsplus;
+            const double HmAp1hm1h = -HpAm1hp1h;
+
+            // A amplitudes for cvl,cvr only: cf. translated to b->c case from [BFvD2014] eq. 3.16
+            result.a_perp_1_L = std::sqrt(2) * N * c910pL * HpVm1hp1h;
+            result.a_para_1_L = -std::sqrt(2) * N * c910mL * HpAm1hp1h;
+            result.a_perp_0_L = std::sqrt(2) * N * c910pL * H0Vp1hp1h;
+            result.a_para_0_L = -std::sqrt(2) * N * c910mL * H0Ap1hp1h;
+
```
Comment (dvandyk) [inline eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc]: you should probably initialise the rest of the amplitudes to zero then.

File: eos/rare-b-decays/lambdab-to-lambdac-l-nu.hh [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+            double differential_a_fb_hadronic(const double & s) const;
+            double differential_a_fb_combined(const double & s) const;
+            double differential_fzero(const double & s) const;
+
+            double integrated_branching_ratio(const double & s_min, const double & s_max) const;
+            double integrated_a_fb_leptonic(const double & s_min, const double & s_max) const;
+            double integrated_a_fb_hadronic(const double & s_min, const double & s_max) const;
+            double integrated_a_fb_combined(const double & s_min, const double & s_max) const;
+            double integrated_fzero(const double & s_min, const double & s_max) const;
+        /*!
```
Comment (dvandyk) [inline eos/rare-b-decays/lambdab-to-lambdac-l-nu.hh]: This is not properly indented, compared to the rest.

File: eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc [hunk 7]
```diff
@@ -0,0 +1,383 @@
...
+        {
+            return lambdab_to_lambdac_l_nu::AngularObservables(this->amplitudes(s))._k;
+        }
+
+        // define below integrated observables in generic form
+        std::array<double, 10> _integrated_angular_observables(const double & s_min, const double & s_max)
+        {
+            std::function<std::array<double, 10> (const double &)> integrand(std::bind(&Implementation::_differential_angular_observables, this, std::placeholders::_1));
+
+            return integrate1D(integrand, 10, s_min, s_max);
```
Comment (dvandyk) [inline eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc]: the 2nd argument needs to be a power of 2. I suggest 32.

**Comment by akokulu7** [comment]
@dvandyk  requested/discussed changes are implemented, could you please review and merge this updated version of the PR? thanks!

**Comment by akokulu7** [comment]
@dvandyk could you please review this updated version of the PR and merge afterwards? thanks.

**Pull request merged**